pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java

```diff
@@ -484,6 +485,19 @@ void run() throws PulsarAdminException {
         }
     }
 
+    @Parameters(commandDescription = "Truncate a topic. \n"
+            + "\t\tThe topic will be truncate, but the latest ledger cannot be deleted.")
```

Comment:
"The topic will be truncated, but the latest ledger cannot be deleted"

Reply:
Thank you very much, I will fix it as soon as possible.